contrib/opencl/Dockerfile

```
@@ -0,0 +1,47 @@
+FROM rust:latest as build
```

```suggestion
FROM ubuntu:23.04 as build
```

Since you're not using rust from `rust:latest` anyway